src/os/host-start.r

```diff
@@ -678,3 +628,66 @@ comment [
     return :host-repl
 ]
 
+
+; Define REPL! object for skinning - stub for elsewhere?
+;
+
+repl!: make object! [
+    repl: true
+    counter: 0
+    last-result: _ ;-- last evaluated result (sent by HOST-REPL)
+
+    ; Called on every line of input by HOST-REPL in %os/host-repl.r
+    ;
+    cycle: does [
+        if zero? ++ counter [print-greeting] ;-- only load skin on first cycle
+        counter
+    ]
+
+    ;; APPEARANCE (can be overridden)
+
+    prompt:   {>> }
+    result:   {== }
+    warning:  {!! }
+    error:    {** }                ;; not used yet
+    info:     to-string #{e29398}  ;; info sign!
+    greeting: _
+    print-prompt:   proc []  [print/only prompt]
+    print-result:   proc []  [print unspaced [result last-result]]
+    print-warning:  proc [s] [print unspaced [warning reduce s]]
+    print-error:    proc [e] [print e]
+    print-info:     proc [s] [print [info space space reduce s]]
+    print-greeting: proc []  [boot-print greeting]
+    print-gap:      proc []  [print-newline]
+
+    ;; BEHAVIOR (can be overridden)
+    
+    input-hook: func [
+        {Receives line input, parse/transform, send back to repl eval}
+        s
+    ][
+        s
+    ]
+
+    dialect-hook: func [
+        {Receives code block, parse/transform, send back to repl eval}
+        s
+    ][
+        s
```

Comment:
Something I want to get feedback on here, is that you don't need to say **:s**, and some of the reasoning for why that is:

https://github.com/metaeducation/ren-c/issues/253

This is a change from Rebol2 and R3-Alpha, and I wonder how people feel about it.